src/main/java/org/gridsuite/modification/server/entities/equipment/modification/MoveVoltageLevelFeederBaysEntity.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.entities.equipment.modification;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import org.gridsuite.modification.dto.ModificationInfos;
+import org.gridsuite.modification.dto.MoveVoltageLevelFeederBaysInfos;
+import org.gridsuite.modification.server.entities.ModificationEntity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Etienne Lesot <etienne.lesot at rte-france.com>
+ */
+@NoArgsConstructor
+@Getter
+@Entity
+@Table(name = "move_voltage_level_feeder_bays")
+@PrimaryKeyJoinColumn(name = "id", foreignKey = @ForeignKey(name = "movefeederbayModification_id_fk_constraint"))
+public class MoveVoltageLevelFeederBaysEntity extends ModificationEntity {
+
+    @Column(name = "voltageLevelId")
```

Comment:
```suggestion
    @Column(name = "voltage_level_id")
```
It is not useful if you precise an explicit name then liquibase transform to another string avec '_'